pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -60,21 +60,18 @@ let
 in
 buildPythonPackage rec {
   pname = "tokenizers";
-  version = "0.13.3";
+  version = "0.14.0";
+  format = "pyproject";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "huggingface";
     repo = pname;
-    rev = "python-v${version}";
-    hash = "sha256-QZG5jmr3vbyQs4mVBjwVDR31O66dUM+p39R0htJ1umk=";
+    rev = "v${version}";
+    hash = "sha256-zCpKNMzIdQ0lLWdn4cENtBEMTA7+fg+N6wQGvio9llE=";
   };
 
-  postPatch = ''
-    ln -s ${./Cargo.lock} Cargo.lock
```

Comment:
if we don't need then cargo.lock anymore, should we remove the file completely from the filesystem ?

Reply:
importCargoLock uses it:

```
  cargoDeps = rustPlatform.importCargoLock {
    lockFile = ./Cargo.lock;
  };
```

Reply:
Missed this, thank you.
If you can just change the commit descriptions then, we wait for ofborg confirmation and this should be good to go.